paymentsheet/api/paymentsheet.api

```diff
@@ -536,6 +536,9 @@ public final class com/stripe/android/paymentsheet/PaymentSheet$InitializationMo
 	public synthetic fun newArray (I)[Ljava/lang/Object;
 }
 
+public final class com/stripe/android/paymentsheet/PaymentSheet$IntentConfiguration$Companion {
```

Comment:
Can we remove this?

Reply:
Talked offline and ignoring this comment.